code/src/components/TotalTodos.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+const Image = styled.img`
+  height: 200px;
+`
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  background-color: tomato;
+  color: white;
+  font-size: 24px;
+`
+const Title = styled.div`
+
+`
+const Total = styled.p`
+
+`
+
+export const TotalTodos = () => {
+  const totalTodos = useSelector((store) => store.tasks.items.length)
+
+return (
+  <Container>
+    <Image src="https://www.svgrepo.com/show/263286/frog.svg" alt="image toad"/>
+    <Title>toado list</Title>
+    <Total>{totalTodos} left</Total>
```

Comment:
This might be a bit picky. Talking from a user perspective: When I read "3 left" I think "3 left to do" and I would expect the number to change as soon as I mark a todo as completed. In fact, the number only changes if I delete a todo. So something like "Currently 3 todos" or just "3 todos" would be more accurate from my point of view.